rgw/v2/tests/s3_swift/configs/test_object_lock.yaml

```diff
@@ -0,0 +1,12 @@
+# upload type: non multipart
+# script: test_object_lock.py
+config:
+ user_count: 1
+ bucket_count: 1
+ objects_count: 2
+ objects_size_range:
+  min: 5
+  max: 15
+ test_ops:
+      create_bucket: true
+      create_object: true
```

Comment:
It is a good YAML practice to have two spacing or a consistent spacing in the file. By this way, we can avoid any loading issues... here we could do
```
config:
  user_count: 1
  bucket_count: 1
  objects_count: 2
  objects_size_range:
    min: 5
    max: 15
  test_ops:
    create_bucket: true
    create_object: true
```

Reply:
Done